org.keepassxc.KeePassXC.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "org.keepassxc.KeePassXC",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "keepassxc",
+  "rename-icon": "keepassxc",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Wayland access */
+    "--socket=wayland",
```

Comment:
We've been disabling Qt's Wayland support as it has issues: https://github.com/flathub/org.telegram.desktop/blob/master/org.telegram.desktop.json#L21-L22

Reply:
Right, that sound familiar. I will simply remove that then.